### ISSUE TITLE
Toggle summary may be truncated

### ISSUE THREAD

**Issue by smor** [souvikinator/notion-to-md#46]
Hello,

I'm trying to convert a Notion `/toggle`'s summary properly, but it is truncated.

This : 
<img width="717" alt="image" src="https://user-images.githubusercontent.com/39006/181475439-56af9ee5-02cb-4c1c-adb9-8763a2d70468.png">
is converted as :
```html
```

**Comment by souvikinator** [comment]
Correct me if I got your requirement wrong. 
You want to render the annotations as well in the toggle summary ?

**Comment by smor** [comment]
IMHO the toggle summary should be treated like any other paragraph and be rendered fully with annotations indeed, links, etc. The difficulty lies in keeping a functional Markdown output along with HTML tags required to produce the toggle, right ?

**Comment by souvikinator** [comment]
True, maybe I should treat toggle as paragraph, but I'm sure it's going to break a lot of websites using this package. Any preferred format you have in mind?

I tried using `setCustomTransformer` like so:
```js
n2m.setCustomTransformer("toggle", async (block) => {
  const { toggle } = block as any;
  // console.log(toggle);
  let toggle_text = "";
  toggle.rich_text.forEach((rich_text: any) => {
    toggle_text += n2m.annotatePlainText(
      rich_text.plain_text,
      rich_text.annotations
    );
  });
  return toggle_text;
});
```
output:
```markdown
This is a <u>~~**toggle**~~</u>
```

however this won't deal with the toggle children.

**Comment by GorvGoyl** [comment]
> Directly producing HTML is not this package's purpose
> but outputting <details> and <summary> is probably the only way this can be done.

Remember HTML is a also valid markdown :)

**Comment by souvikinator** [comment]
@GorvGoyl @smor Gents there have been a lot of customization related issues in the library and it's limitation to MD. 
I have opened a github discussion with proposal for V4 addressing these issues. Please check out #112 

It'll be a perfect time to have your invaluable feedback on it :)
